src/test/resources/data/1000Eori.json

```diff
@@ -0,0 +1,1006 @@
+{
+  "date": "2024-02-08",
+  "authType": "UKNW",
+  "eoris": [
+    "GB769039790992536",
```

Comment:
These all contain the same number. It's not really a reflection of a valid request.

Reply:
Changed the requests to contain different EORI